.github/workflows/static_checks.yml

```diff
@@ -24,8 +24,8 @@ jobs:
 
       - name: Install dependencies
         run: |
-          sudo apt-get install -qq dos2unix recode clang-format-13 libxml2-utils
-          sudo update-alternatives --remove-all clang-format
+          sudo apt-get install -qq dos2unix recode clang-format-13 libxml2-utils python3-pip moreutils
```

Comment:
Btw, `recode` can be removed from here, and removed from the pre-loop checks in file_format.sh.

Reply:
Good catch, done.